sdk/maps/test-resources.bicep

```diff
@@ -0,0 +1,44 @@
+param baseName string = resourceGroup().name
+param location string = resourceGroup().location
+param testApplicationOid string
+
+var roleDefinitionId = '8f5e0ce6-4f7b-4dcf-bddf-e6f48634a204'
```

Comment:
Can we add a comment here so we know this is "Azure Maps Data Contributor" roles?